pkg/kubefed2/unjoin.go

```diff
@@ -65,6 +64,8 @@ func (o *unjoinFederationOptions) Bind(flags *pflag.FlagSet) {
 		"Name of the cluster's context in the local kubeconfig. Defaults to cluster name if unspecified.")
 	flags.BoolVar(&o.removeFromRegistry, "remove-from-registry", false,
 		"Remove the cluster from the cluster registry running in the host cluster context.")
+	flags.BoolVar(&o.forceDeletion, "force", false,
+		"Forcely delete federeated cluster and secret resources for the unjoined cluster.")
```

Comment:
`Delete federated cluster and secret resources even if resources in the cluster targeted for unjoin are not removed successfully`